src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -41,16 +41,20 @@ public CommandResult execute() {
     }
 
     /**
-     * Retrieves all persons in the address book whose names contain some of the specified keywords.
-     *
+     * Retrieves all persons in the address book whose names contain
+     * some of the specified keywords, regardless of casing.
      * @param keywords for searching
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
+        ArrayList<String> lowerKeywords = new ArrayList<>();
+        for (String word : keywords) {
```

Comment:
FYI: There's a neater way to do this. [See here.](https://dzone.com/articles/how-to-use-map-filter-collect-of-stream-in-java-8)